### PR TITLE
refactor!: `CardMedium`, `CardLarge` move to layout package

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
Closes #

### CODE REVIEW

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts [hunk 1]
```diff
@@ -345,4 +345,8 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifier[] = [
         from: {name: 'TuiCellModule', moduleSpecifier: '@taiga-ui/experimental'},
         to: {name: 'TuiCellDirective', moduleSpecifier: '@taiga-ui/layout'},
     },
+    {
+        from: {name: 'TuiCardModule', moduleSpecifier: '@taiga-ui/experimental'},
+        to: {name: 'TuiCardModule', moduleSpecifier: '@taiga-ui/layout'},
```
Comment (splincode) [inline projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts:350]: TuiCardLargeDirective?

File: projects/demo/src/modules/components/card-large/examples/import/import.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+```ts
+import {Component} from '@angular/core';
+import {TuiCardLargeDirective} from '@taiga-ui/layout';
+// ...
+
+@Component({
+  standalone: true,
+  imports: [
+    // ...
+    TuiCardModule,
```
Comment (splincode) [inline projects/demo/src/modules/components/card-large/examples/import/import.md:10]: TuiCardLargeDirective?

Comment (splincode) [inline projects/demo/src/modules/components/card-large/examples/import/import.md:10]: Drop TuiCardModule

File: projects/cdk/schematics/ng-update/steps/replace-identifier.ts [hunk 3]
```diff
@@ -38,13 +38,29 @@ export function replaceIdentifier({from, to}: ReplacementIdentifier): void {
...
+            addUniqueImport(filePath, namedImport || name, moduleSpecifier);
+        });
+    } else {
+        addUniqueImport(
+            filePath,
+            identifier.namedImport || identifier.name,
+            identifier.moduleSpecifier,
+        );
+    }
+}
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/steps/replace-identifier.ts:66]:
```suggestion
function addImports(
    identifier: ReplacementIdentifierMulti['to'],
    filePath: string,
): void {
    return Array.isArray(identifier)
        ? identifier.forEach(x => addImports(x, filePath))
        : addUniqueImport(
              filePath,
              identifier.namedImport || identifier.name,
              identifier.moduleSpecifier,
          );
}
```

What do you think about this approach ? 🤔

**Pull request merged**